snapcraft.yaml

```diff
@@ -247,12 +253,14 @@ parts:
       - lib/*/librtmp.so*
       - lib/*/libsasl2.so*
       - lib/*/libsnappy.so*
-      - lib/*/libncurses.so*
+      - lib/*/libssh.so*
+      - lib/*/libtcmalloc.so*
+      - lib/*/libunwind.so*
 
   criu:
     source: https://github.com/checkpoint-restore/criu
-    source-depth: 1
-    source-tag: v3.19
+    source-commit: f8b14286b092853a4485813e1efd564109df9123 # v3.19
+    #source-depth: 1
```

Comment:
The `source-depth` and git stuff for the `criu` part seems to be for `core24` only so you might want to drop those, according to the PR description.

Reply:
it builds ok on core22, and if https://github.com/canonical/lxd-pkg-snap/pull/623 builds ok on armhf ill bump this up to v4 and remove the patches anyway.